src/mqueryfront/src/api.js

```diff
@@ -9,14 +9,21 @@ export function parseJWT(token) {
 }
 
 function request(method, path, payload, params) {
-    const rawToken = localStorage.getItem("token");
+    const rawToken = localStorage.getItem("rawToken");
     const headers = rawToken ? { Authorization: `Bearer ${rawToken}` } : {};
-    return axios.request(path, {
-        method: method,
-        data: payload,
-        params: params,
-        headers: headers,
-    });
+    return axios
+        .request(path, {
+            method: method,
+            data: payload,
+            params: params,
+            headers: headers,
+        })
+        .catch((error) => {
+            if (error.response.status === 401) {
+                window.location = "/auth";
```

Comment:
This won't be annoying where a user tries to do some action they don't have permission to do?

Reply:
Ideally the web UI will hide all the actions that the client is not allowed to do, so this shouldn't be annoying.

Also 401 is only when user is not logged in - when user is logged in, they get 403 which is just a regular red-error-bar error